backend/src/main/java/com/bakdata/conquery/resources/api/ConceptsProcessor.java

```diff
@@ -220,7 +220,11 @@ public AutoCompleteResult autocompleteTextFilter(SelectFilter<?> filter, Optiona
 			out.addAll(result);
 		}
 
-		return out;
+		// The different sources might contain duplicate FEValue#values which we want to avoid
+		// They are already sorted in terms of information weight
+		return out.stream()
```

Comment:
Wie passiert es, dass es da zu Doppelungen kommt?

Reply:
FEValue ist nur über den value distinct. Wenn ich in den Daten einen Wert habe, will ich ja immer das Mapping dafür haben, der rohe wert ist eher uninteressant, hier werden die dedupliziert. Doku-Kommentar kann ich noch nachbessern.